### PR TITLE
fix: only print controller comment for Go projects

### PR DESCRIPTION
**Pull request by jmrodri**
**Description of the change:**
During conversion of 3-alpha config, only print out controller comment if it is a Go project.

**Motivation for the change:**
The controller field is not relevant to Helm and Ansible projects.

**Checklist**

If the pull request includes user-facing changes, extra documentation is required:
- [ ] Add a new changelog fragment in `changelog/fragments` (see [`changelog/fragments/00-template.yaml`](https://github.com/operator-framework/operator-sdk/tree/master/changelog/fragments/00-template.yaml))
- [ ] Add or update relevant sections of the docs website in [`website/content/en/docs`](https://github.com/operator-framework/operator-sdk/tree/master/website/content/en/docs)

### CODE REVIEW

**Comment by jmrodri** [comment]
Fixes #4711

**Review by camilamacedo86 — APPROVED** [review]
/lgtm

File: internal/cmd/operator-sdk/cli/cli.go.orig [hunk 1]
```diff
@@ -1,149 +0,0 @@
-// Copyright 2019 The Operator-SDK Authors
```
Comment (camilamacedo86) [inline internal/cmd/operator-sdk/cli/cli.go.orig:1]: This should not get merged by sure :-P

**Review by estroz — APPROVED** [review]
/lgtm

**Pull request merged**